### PR TITLE
[firebase_crashlytics] Updated the documentation language.

### PR DESCRIPTION
**Pull request by feinstein**
The english was a bit confusing.

## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] If the pull request affects only one plugin, the PR title starts with the name of the plugin in brackets (e.g. [cloud_firestore])
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by collinjackson** [comment]
Looks good, can you please update CHANGELOG and pubspec.yaml (this is a +1 version bump)

**Comment by feinstein** [comment]
@collinjackson done.

File: packages/firebase_crashlytics/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.1.2+1
+
+* Updated the docs.
```
Comment (collinjackson) [inline packages/firebase_crashlytics/CHANGELOG.md]:
```suggestion
* Updated a confusing comment.
```

**Comment by creativecreatorormaybenot** [comment]
@feinstein Thanks for updating my comment. Can you explain to me what about it was confusing exactly? I tried to make it more clear by repeating the part before the colon, but I am not confident in my langauge, hence, it would be amazing to learn here.

**Comment by feinstein** [comment]
There shouldn't be a repetition, that's not proper grammar. The reader already know it, so the language should flow as a continuity. You can see this kind of language is books and technical documentes as well.

**Comment by creativecreatorormaybenot** [comment]
@feinstein Thank you.

**Comment by feinstein** [comment]
My pleasure :)

**Pull request merged**